pkg/labels/labels.go

```diff
@@ -0,0 +1,61 @@
+/*
+Copyright 2023 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package labels
+
+import (
+	"strings"
+)
+
+// Tags is a slice of string. It implements Labels.
+type Tags []string
+
+// Has returns whether the provided label exists in the slice.
+func (ls Tags) Has(label string) bool {
+	for _, tag := range ls {
+		if !strings.HasPrefix(tag, "caphvlabel:") {
+			continue
+		}
+
+		trimmedTag := strings.TrimPrefix(tag, "caphvlabel:")
+		splittedTrimTag := strings.Split(trimmedTag, "=")
+
+		if len(splittedTrimTag) == 2 && splittedTrimTag[0] == label {
```

Comment:
return len(...) == 2 ...
without if else

Reply:
oh sorry, I just realized that this is inside the for loop. Then it doesn't make sense and the previous approach was correct right?

Reply:
ahh yes otherwise we want to ignore that

Reply:
reverted